### DISCUSSION TITLE
教务系统登陆失败

### DISCUSSION THREAD

**Discussion by Ao-Wu-Benjamin** [IceTiki/ruoli-sign-optimization#133] [Q&A]
![image](https://user-images.githubusercontent.com/102411395/196028061-998cf119-f7d3-40a2-b231-b537b4fcbf87.png)

**Comment by IceTiki** [comment]
可能是你网不好，或者服务器关掉了，你试试在签到时段登陆下你的教务系统试试

**Comment by Ao-Wu-Benjamin** [comment]
不是网络问题，本地打卡是可以的。但是云上就不行了

**Comment by IceTiki** [comment]
> 不是网络问题，本地打卡是可以的。但是云上就不行了

重试也不行吗，还是你用了国外的节点
